app/controllers/cards_controller.rb

```diff
@@ -3,6 +3,20 @@ def index
     @cards = Card.all
   end
 
+  def edit
+    @card = Card.find(params[:id])
+  end
+
+  def update
+    @card = Card.find(params[:id])
+
+    if @card.update(cards_params)
+      redirect_to cards_path
+    else
+      render 'edit'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.